strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/services/impl/ConfigurationManagementServiceImpl.java

```diff
@@ -618,4 +624,38 @@ private void modifyInLock(final Consumer<MutableConfiguration> operation,
         }
     }
 
+    @Override
+    public boolean setMaxUploadFileSize(String maxFileSizeString) throws SizeFormatValidationException
+    {
+
+        if(this.multipartConfigElement instanceof UpdatableMultipartConfigElement){
+            Long parsedSize = parseUploadSize(maxFileSizeString);
+            ((UpdatableMultipartConfigElement) multipartConfigElement).setMaxFileSize(parsedSize);
+            return true;
+        }
```

Comment:
Cosmetic: Coding convention, braces on new line.
```suggestion
        if (this.multipartConfigElement instanceof UpdatableMultipartConfigElement)
        {
            Long parsedSize = parseUploadSize(maxFileSizeString);
            ((UpdatableMultipartConfigElement) multipartConfigElement).setMaxFileSize(parsedSize);
            
            return true;
        }
```